lib/backpex/fields/select.ex

```diff
@@ -1,11 +1,24 @@
 defmodule Backpex.Fields.Select do
+  @config_schema [
+    options: [
+      doc: "List of options or function that receives the assigns.",
+      type: {:or, [:keyword_list, {:fun, 1}]},
```

Comment:
`options` may also be a list of tuples, e.g. `[{"Admin", "admin"}]`. I think a list of strings can also be used. In the future, we may also have to support groups (like described here: https://hexdocs.pm/phoenix_html/Phoenix.HTML.Form.html#options_for_select/2)

Maybe we just use `{:list, :any}` instead of `:keyword_list` for now? 

Otherwise, we should allow all options that are allowed for `options_for_select/2`. See https://github.com/phoenixframework/phoenix_html/blob/v4.1.1/lib/phoenix_html/form.ex#L303